modules/configuration/pages/secrets.adoc

```diff
@@ -1,26 +1,68 @@
 = Secrets
+description: Methods for adding secrets to your Redpanda configuration without exposing them.
 
 This topic outlines how to add secrets to a Redpanda Connect configuration without exposing them.
 
-== Using environment variables
+== Store secrets in environment variables
```

Comment:
@Jeffail Could you check changes to this section are accurate?

Reply:
LGTM